dlrs/main/classes/RollupCalculateJob.cls

```diff
@@ -113,19 +113,31 @@ public with sharing class RollupCalculateJob implements Database.Batchable<sObje
       // Retrieve organization info to put in the email
       Organization org = [SELECT Id, Name FROM Organization];
 
+      //Getting rollup name
+      String rollupName = rollup.isEmpty() ? '' : rollup[0].Name;
```

Comment:
Thank you for covering this case where we might not have a rollup that matches. Were you able to test this case?